_includes/js/objects.md

```diff
@@ -89,6 +89,36 @@ However, when using `extends`, the SDK is not automatically aware of your subcla
 Parse.Object.registerSubclass('Monster', Monster);
 ```
 
+Similarly, you can `extends` `Parse.User`
```

Comment:
Maybe this should say "Similarly, you can use `extends` with `Parse.User`"

Reply:
Sounds good